Enhance abstract skill for DITA short descriptions (#4814)
theforeman/foreman-documentation: .cursor/skills/abstract/references/procedure.md

# Abstract instructions: procedure modules

> When the module has `:_mod-docs-content-type: PROCEDURE`, follow these rules for the abstract (short description):

Reviewer: Same as for concept modules

Author: ```suggestion
When the module is a procedure module (is named `proc_*.adoc` or includes `:_mod-docs-content-type: PROCEDURE` as its first line), follow these rules for the abstract (short description):
```

- Use phrases such as "You can [action] to [benefit]", "To [goal], configure [feature]", or "[Action] [what] to [why]".
- Explain the task that users can accomplish, the benefits of the task, and the purpose of the task.

> - Do not simply repeat the title.

Reviewer: ```suggestion
```
This duplicates one of the lines in the abstract skill file.

> - Include information that will help users understand when the task is appropriate or necessary.

Reviewer: ```suggestion
- Explain the task that users can accomplish, the benefits of the task, and the purpose of the task.
- Do not simply repeat the title.
- Include information that will help users understand when the task is appropriate or necessary.
- Use phrases such as "You can [action] to [benefit]", "To [goal], configure [feature]", or "[Action] [what] to [why]".
```

> - When the abstract is too short, thin, or generic, add more information about why the task is beneficial.

Reviewer: What does "thin" mean? Can you find a more precise word to describe what you mean?

Author: I'll just drop it

Author: ```suggestion
- When the abstract is too short or generic, add more information about why the task is beneficial.
```
